tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"esModuleInterop": true,
+		"isolatedModules": true,
+		"jsx": "react-jsx",
+		"lib": [ "dom", "esnext" ],
+		"module": "esnext",
+		"moduleResolution": "node",
+		"noEmit": true,
+		"noFallthroughCasesInSwitch": true,
+		"resolveJsonModule": true,
+		"skipLibCheck": true,
+		"strict": false,
+		"target": "ES6",
+		"paths": {
+			// e.g. "@automattic/jetpack-components"
+			// will resolve to "projects/js-packages/components"
+			"@automattic/jetpack-*": [ "./projects/js-packages/*" ]
+		}
+	},
+	"include": [ "projects/**/*", "tools/**/*" ]
```

Comment:
This also seems broken. We shouldn't need to be including every file in every project, especially not here.

Reply:
Removed.